cime_config/acme/machines/template.lt_archive

```diff
@@ -1,177 +1,117 @@
+    os.environ["maxthrds"] = 1
+
+    # document start
+    if os.path.isfile(testlog):
+        with open(testlog, "a") as fd:
+            fd.write("lt_archive starting %s" %(time.strftime("%Y-%m-%d %H:%M:%S")))
+    logger.info("lt_archive starting %s" %(time.strftime("%Y-%m-%d %H:%M:%S")))
+
+    # determine status of run and short term archiving
+    runComplete = False
+    statusfile = os.path.join(caseroot,"CaseStatus") 
+    if os.path.isfile(statusFile):
+        if 'run SUCCESSFUL' in open(statusFile).read():
+            runComplete = True
+
+    statusComplete = False
+    statusfile = os.path.join(caseroot,"stArchiveStatus")
+    if os.path.isfile(statusFile):
+        if 'st_archive process complete' in open(statusFile).read():
+            runComplete = True
 
     # set up envrionment vars and call the lt_archive.sh script
-    my $runComplete = checkRun( "$config{'CASEROOT'}/CaseStatus" );
-    my $staComplete = checkSta( "$config{'CASEROOT'}/stArchiveStatus" );
-
-    if( $runComplete && $staComplete )
-    {
-	$ENV{DOUT_S_ROOT} = $config{'DOUT_S_ROOT'};
-	$ENV{DOUT_L_MSROOT} = $config{'DOUT_L_MSROOT'};
-	$ENV{DOUT_L_HPSS_ACCNT} = $config{'DOUT_L_HPSS_ACCNT'};
-
-	my $LID = strftime("%y%m%d-%H%M%S", localtime);
-
-	qx($config{'CASEROOT'}/Tools/lt_archive.sh {{ lt_archive_args }} >> ltArchiveStatus.$LID 2>&1);
-    }
-    else
-    {
-	$logger->info("lt_archive: run or st_archive is not yet complete or was not successful. Unable to perform long term archive...");
-	exit 1;
-    }
-
-    if(-e $testlog)
-    {
-        open(TL, ">>$testlog");
-        print TL "lt_archive complete\n";
-        close(TL);
-    }
-    $logger->info("lt_archive complete\n");
-    resubmitCheck();
-}
-
-#-----------------------------------------------------------------------------------------------
-# resubmitCheck - check if the case needs to be resubmitted
-#-----------------------------------------------------------------------------------------------
-
-sub resubmitCheck()
-{
-    if($config{RESUBMIT} > 0)
-    {
-        my $submitscript = "$config{CASEROOT}/$config{CASE}.submit";
-        my $cwd = getcwd;
-        chdir $config{CASEROOT};
-        my $resubmitcommand = "$submitscript -resubmit -scriptname $config{'CASE'}.lt_archive";
-        $logger->debug(" running resubmit check $resubmitcommand");
-        if(-e $testlog)
-        {
-            open(TL, ">>$testlog");
-            print TL "lt_archive complete\n";
-            close(TL);
-        }
-        qx($resubmitcommand);
-        if($?) { $logger->warn("could not run CESM submit script, $! $?") ; }
-        chdir $cwd;
-    }
-}
-main(@ARGV) unless caller;
+    if runComplete and staComplete:
+        os.environ["DOUT_S_ROOT"]       = case.get_value("DOUT_S_ROOT")
+        os.environ["DOUT_L_MSROOT"]     = case.get_value("DOUT_L_MSROOT")
+	os.environ["DOUT_L_HPSS_ACCNT"] = case.get_value("DOUT_L_HPSS_ACCNT")
+
+	lid = time.strftime("%y%m%d-%H%M%S")
```

Comment:
This is odd. My script does not show this. I've downloaded pylint and will use this in the future.